bitnami/jupyterhub/templates/image-puller/daemonset.yaml

```
@@ -68,6 +68,9 @@ spec:
         - name: pull-{{ $index }}
           image: {{ include "common.images.image" (dict "imageRoot" . "global" $.Values.global) }}
           imagePullPolicy: {{ .pullPolicy }}
+          {{- if .Values.imagePuller.resources }}
+          resources: {{- toYaml .Values.imagePuller.resources | nindent 12 }}
+          {{- end }}
```

Hi, this won't work.
Could you try this ?
```suggestion
          {{- if $.Values.imagePuller.resources }}
          resources: {{- toYaml $.Values.imagePuller.resources | nindent 12 }}
          {{- end }}
```

---

Thanks for the hint. I have changed it.